src/dattobd.c

```diff
@@ -4121,13 +4121,95 @@ static int ioctl_dattobd_info(struct dattobd_info *info){
 	return ret;
 }
 
+static int ioctl_dattobd_active_device_info(struct dattobd_active_device_info **adinfo, unsigned int *copyback)
+{
+    /* get the count from *info, free the memory, figure out how much we're actually going to return
+     * reallocate *info to that size, then copy the structs in, set the count for what we're returning
+     * and we're done. */
```

Comment:
I don't get what you mean by the multi-line comment.
the comment variable name is wrong, because I changed the variable name a few times, it used to be info. :-)